### ISSUE TITLE
Accessibility - Goods List Screen - TalkBack Reading is Incorrect 

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#41]
After reading what should be the heading screen TalkBack reads the first good after the section heading (needs to be heading) and then the section heading followed by the next country. Same thing happens with the other sections too and the same for other sections on the screen. Not sure why the content is being read out of order when it should be read from top to bottom then left to right.

The problem is for all the different tabs on this screen (A –Z and By Sector).

**Comment by aquilent-dgemoets** [comment]
See discussion for #32.

**Comment by jfc3-dol** [comment]
I guess, since you removed/changed third party widgets to create the goods lists in the different sections it makes it a better experience but would be better having headings on the different screen/tabs.

**Comment by jfc3-dol** [comment]
Closing issue.
